notebooks/MAST/plotting_quasar_observations/plotting_quasar_observations.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>On the output it appears to make a new plot instead of adding to the old plot, bummer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>This is just the answer to the exercise for the user. So they are supposed to add it to the plot above themselves if they figure it out.</p>